tools/python/maps_generator/README.md

```diff
@@ -228,26 +249,29 @@ DEBUG: 0
 PLANET_URL: https://download.geofabrik.de/asia/japan-latest.osm.pbf
 PLANET_MD5_URL: https://download.geofabrik.de/asia/japan-latest.osm.pbf.md5
 ...
+
+To build an entire country with coastlines, you need to download the *latest_coasts.geom* and *latest_coasts.rawgeom* files and specify their path in the config.
```

Comment:
It is only needed in the case of building some regions. If the whole planet is built, then these files will be generated/used directly.

Reply:
Are you saying that this part of the config can be left blank in a planet build? For this example, we are doing all of Japan. 

I have found that the generator fails if I include World and WorldCoasts, at least for the Yukon. So aren't the geom files always required?

Reply:
You don't need to download these files if the whole planet is built. They are generated in the process of building the whole planet.